packages/@okta/vuepress-theme-prose/package.json

```diff
@@ -27,15 +27,13 @@
     "@okta/okta-signin-widget": "^5.4.0",
     "axios": "^0.21.0",
     "cookies.js": "^2.1.15",
-    "copy-webpack-plugin": "^8.1.0",
```

Comment:
This wasn't needed - In fact causing errors because the version of copy in vuepress is 5.1 and this causes the overall build to try to use 8.1 and created build errors because it was trying to read the parameters meant for a v5.1 plugin.